src/clusterfuzz/_internal/bot/fuzzers/centipede/engine.py

```diff
@@ -105,12 +105,25 @@ def prepare(self, corpus_dir, target_path, build_dir):
     # Directory corpus_dir saves the corpus files required by ClusterFuzz.
     arguments.append(f'--corpus_dir={corpus_dir}')
 
-    # The unsanitized binary, Centipede requires it to be the main fuzz target.
-    arguments.append(f'--binary={target_path}')
-
+    # Centipede requires at least one of:
+    # 1. The unsanitized target binary.
+    # 2. The sanitized target binary.
+    unsanitized_target_path = pathlib.Path(target_path)
+    unsan_bin_exists = unsanitized_target_path.exists()
```

Comment:
Instead of switching to calling it san/unsan, let's be consistent and call it sanitized/unsanitized.

Reply:
BTW, I may also need to change to config to always download sanitized binaries to the `extra_build` directory, instead of the default directory (which is for unsanitized binary).
Is it possible to do it in job config, or should I modify the code in clusterfuzz?